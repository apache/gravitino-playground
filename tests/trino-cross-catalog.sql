SELECT given_name, family_name, job_title, sum(total_amount) AS total_sales
FROM gt_hive2.sales.sales as s,
  catalog_postgres.hr.employees AS e
where s.employee_id = e.employee_id
GROUP BY given_name, family_name, job_title
ORDER BY total_sales DESC
LIMIT 1;

SELECT customer_name, location, SUM(total_amount) AS total_spent
FROM gt_hive2.sales.sales AS s,
  gt_hive2.sales.stores AS l,
  gt_hive2.sales.customers AS c
WHERE s.store_id = l.store_id AND s.customer_id = c.customer_id
GROUP BY location, customer_name
ORDER BY location, SUM(total_amount) DESC;

SELECT e.employee_id, given_name, family_name, AVG(rating) AS average_rating, SUM(total_amount) AS total_sales
FROM catalog_postgres.hr.employees AS e,
  catalog_postgres.hr.employee_performance AS p,
  gt_hive2.sales.sales AS s
WHERE e.employee_id = p.employee_id AND p.employee_id = s.employee_id
GROUP BY e.employee_id,  given_name, family_name;