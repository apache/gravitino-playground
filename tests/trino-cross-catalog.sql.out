"10","Chelsea","Wade","5.0","299.97"
"11","Clarke","Sanders","5.833333333333333","3329.34"
"13","Risa","Barber","7.0","1779.72"
"15","Oprah","Noel","5.0","1959.82"
"18","Carolyn","Bradshaw","7.0","1029.90"
"19","Xyla","Le","4.0","271.92"
"21","Carol","Decker","5.666666666666667","2924.16"
"22","Quemby","Lucas","5.0","897.47"
"23","Phoebe","Forbes","4.25","1919.76"
"25","Elijah","Burnett","3.8","249.95"
"28","Reuben","Rojas","4.25","3799.68"
"29","Maxwell","Patel","7.0","2759.88"
"30","Edward","Reed","5.333333333333333","168.00"
"32","Jesse","Contreras","5.0","1049.76"
"41","Dale","Lindsey","7.0","1809.87"
"42","Maite","Riddle","5.0","485.43"
"44","Perry","Roberson","7.5","49.98"
"46","Oleg","Tran","10.0","1739.91"
"48","Mohammad","Caldwell","6.0","256.47"
"49","Zephr","Long","6.0","179.98"
"52","Laura","Macdonald","7.166666666666667","3839.76"
"6","Jasper","Mack","4.0","2207.64"
"Dale","Lindsey","Sales Assistant","1809.87"
"Erasmus Phelps","Nebraska","1553.79"
"Erasmus Phelps","Texas","169.99"
"Erasmus Phelps","Vermont","199.98"
"Harriet Best","Kansas","4229.70"
"Harriet Best","Nebraska","1039.81"
"Harriet Best","Texas","256.47"
"Harriet Best","Vermont","719.88"
"Lenore Wilder","Kansas","1594.84"
"Lenore Wilder","Nebraska","129.98"
"Mia Hahn","Kansas","94.97"
"Mia Hahn","Nebraska","2479.82"
"Mia Hahn","Texas","279.98"
"Mia Hahn","Vermont","479.88"
"Perry Tyler","Kansas","99.99"
"Perry Tyler","Nebraska","1379.94"
"Perry Tyler","Vermont","299.98"
"Raya Mcguire","Kansas","349.92"
"Raya Mcguire","Nebraska","1674.46"
"Raya Mcguire","Texas","135.98"
